com.github.babluboy.nutty.json

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "com.github.babluboy.nutty",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "nutty",
+    "rename-appdata-file": "com.github.babluboy.nutty.appdata.xml",
```

Comment:
It is already correct, so renaming it doesn't make sense.

Reply:
@TingPing Thanks, removed the rename clause